css/css-logical/logical-longhand-properties.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8" />
+<title>Test for handling of logical and physical properties</title>
+<link rel="author" title="Cameron McCormack" href="mailto:cam@mcc.id.au">
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-logical-1/#box">
```

Comment:
It's always nice to add a `<meta name="assert"` explaining what the test is specifically testing. Could you add it?